packages/boxel-ui/addon/src/components/resizable-panel-group/index.gts

```diff
@@ -124,6 +116,29 @@ export default class ResizablePanelGroup extends Component<Signature> {
     return this.panelGroupElement?.[this.offsetLengthProperty];
   }
 
+  private get panelGroupLengthWithoutResizeHandlerPx() {
+    let resizeHandlerSelector = `#resize-handler-${this.args.orientation}-0`;
+    let resizeHandlerEl = this.panelGroupElement?.querySelector(
+      resizeHandlerSelector,
+    ) as HTMLElement;
+    if (!resizeHandlerEl) {
+      console.error(
+        `Could not find selector: ${resizeHandlerSelector} when handling window resize for resizeable panel group`,
+      );
+      return undefined;
+    }
```

Comment:
I know this is how it’s been for a while but maybe something like [`ember-ref-bucket`](https://github.com/lifeart/ember-ref-bucket) would be a useful modifier to track and pass elements around instead of going through the DOM.

Reply:
Updated [1032fda](https://github.com/cardstack/boxel/pull/821/commits/1032fda4cc233dac3c0db7c2d9ae89bbd462ada3)